recipes/zlib-ng/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "2.0.0":
+    sha256: "86993903527d9b12fc543335c19c1d33a93797b3d4d37648b5addae83679ecd8"
+    url: "https://github.com/Dead2/zlib-ng/archive/2.0.0.tar.gz"
```

Comment:
There's a 2.0.2 https://github.com/zlib-ng/zlib-ng/releases/tag/2.0.2

Reply:
need this version, will then add 2.0.2